src/apps/dashboard/modal/ReservationsGroupModal.tsx

```diff
@@ -32,22 +22,20 @@ import { mapPublizonReservationToReservationType } from "../../../core/utils/hel
 interface ReservationGroupModalProps {
   pageSize: number;
   modalId: string;
+  setReservationsToDelete: (reservations: string[]) => void;
```

Comment:
Please consider if typing and or naming of this can be improved.

To avoid just passing strings around we have types for other ids. Could we create a new type for reservation ids and use this here?

Since we are not expecting `ReservationType` instances here I would also suggest renaming the value to `reservationIds`.